fire_atlas_editor/lib/screens/editor_screen/widgets/selection_canvas/selection_form.dart

```diff
@@ -112,15 +123,24 @@ class _SelectionFormState extends State<SelectionForm> {
 
         return;
       }
+      final frameCount = int.parse(frameCountController.text);
+      final stepTime = int.parse(stepTimeController.text) / 1000;
+
+      final selectionToSave = widget.editingSelection == null
+          ? AnimationSelection(
+              info: _fillSelectionBaseValues(),
+              frameCount: frameCount,
+              stepTime: stepTime,
+              loop: _animationLoop,
+            )
+          : widget.editingSelection!
```

Comment:
`a ?? b`

Reply:
I tried that, but it don't play well with the cascading operator that I need to use a few lines below.